packages/lib/test/src/utils/Transactions.test.js

```diff
@@ -67,6 +73,34 @@ contract('Transactions', function([_account1, account2]) {
       assertGasLt(tx, 1000000);
     });
 
+    it('uses gas price API when gas not specified', async function () {
+        const oldNetwork = state.network
+
+        state.network = "1";
+        const stub = sinon.stub(axios, 'get').resolves({ average: 49 })
```

Comment:
Let's move these two lines to a beforeEach block. If needed, we can wrap these two tests in a "describe" block, so the beforeEach only affects them.